msg/airspeed_validated.msg

```diff
@@ -10,3 +10,5 @@ float32 true_ground_minus_wind_m_s 		# TAS calculated from groundspeed - windspe
 bool airspeed_sensor_measurement_valid 		# True if data from at least one airspeed sensor is declared valid.
 
 int8 selected_airspeed_index 			# 1-3: airspeed sensor index, 0: groundspeed-windspeed, -1: airspeed invalid
+
+# TOPICS airspeed_validated airspeed_smoothed
```

Comment:
It seems weird to me to make an entirely separate topic where only 1 field is defined. Two options IMO:
1. Add a smoothed field to airspeed_validated
2. Define & smooth all fields in the new topic

Reply:
This is a quick way of logging the value during initial test flights. Could probably have made it easier to review by adding an inline comment in addition to the commit message.

Reply:
It doesnt make sense to me that adding an additional topic is easier than modifying an existing one. How is it like that?

Reply:
Because the existing topic is published from a different module.

Reply:
Replaced this with the vtol transition status.